packages/manager/apps/container/src/container/legacy/template.module.scss

```diff
@@ -9,6 +9,7 @@
 
   &_content {
     flex: 1 1 auto;
+    height: 100%; /* For Safari on BigSur or before */
```

Comment:
Release name is `Big Sur` (not `BigSur`) :eyes:.
It might be more interesting to specify which Safari version is causing the issue rather than the OS version.

Reply:
Apparently the problem comes from the OS not the Safari version, as i am the only one who can reproduce it because i am on Catalina.

Reply:
Issue should be related to a given version of Safari, not OS related (because we can upgrade Safari in Big Sur (at least to a given version).